framework/include/vectorpostprocessors/VectorPostprocessorData.h

```diff
@@ -34,8 +34,17 @@ class VectorPostprocessorData : public Restartable
   /// VectorPostprocessorState (2 containers for values (see MooseTypes.h)
   struct VectorPostprocessorState
   {
-    VectorPostprocessorValue * current;
-    VectorPostprocessorValue * old;
+    VectorPostprocessorValue * current = NULL;
```

Comment:
I'd prefer to see `nullptr` for new uses of null.

Reply:
👍